eng/common/scripts/Submit-PullRequest.ps1

```diff
@@ -49,40 +49,64 @@ param(
   [string]$PRBody = $PRTitle,
 
   [Parameter(Mandatory = $false)]
-  [string]$PRLabels
+  [string]$PRLabels,
+
+  [Parameter(Mandatory = $false)]
+  [string]$UserReviewers,
+
+  [Parameter(Mandatory = $false)]
+  [string]$TeamReviewers,
+
+  [Parameter(Mandatory = $false)]
+  [string]$Assignees
 )
 
 $headers = @{
   Authorization = "bearer $AuthToken"
 }
+$baseURI = "https://api.github.com/repos"
+function SplitMembers ($membersString)
+{
+  return @($membersString.Split(",") | % { $_.Trim() } | ? { return $_ })
+}
 
-$query = "state=open&head=${PROwner}:${PRBranch}&base=${BaseBranch}"
+function InvokeGitHubAPI($apiURI, $method, $body) {
+  $resp = Invoke-RestMethod -Method $method -Headers $headers -Body ($body | ConvertTo-Json) -Uri $apiURI -MaximumRetryCount 3
+  Write-Host -f green "These members have been added to: https://github.com/$RepoOwner/$RepoName/pull/$prNumber"
+  ($body | Format-List | Write-Output)
```

Comment:
Why dump the body after the invoke?

Reply:
I wanted a way to log what members were added.

Reply:
OK I didn't see your other console line. I'd suggest you use Write-Host here instead of Write-Output as you probably don't want this information written to the output pipeline but instead just the console.

Reply:
Using Write-Host in this case does not give the correct info.